### PR TITLE
Micropotfolio

### PR DESCRIPTION
**Pull request by obibaadoma**
Changes to Skills and Buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -254,10 +263,10 @@ <h3>2015</h3>
   <section class="contact-me">
     <div class="head-section-co">
       <div id="head-container-co">
-        <h1 class="head-co">
+        <h2 class="head-co">
           About, <br />
           Myself
-        </h1>
+        </h2>
```
Comment (Lizdev-05) [inline index.html]: Great work @obibaadoma  👍, Kindly give this 👆 head tag the correct style.
**Your design**
wrongabt

**Figma design**
abt

File: index.html [hunk 2]
```diff
@@ -330,40 +339,25 @@ <h3 class="languages">Languages</h3>
             <div class="event17">
               <h3>Frameworks</h3>
               <button type="submit" class="dropdown">
-                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                <img src="./images/Disabled.png" alt="" />
```
Comment (Lizdev-05) [inline index.html:336]: Kindly check this image path 😉, the image is not displaying.
img

File: index.html [hunk 3]
```diff
@@ -330,40 +339,25 @@ <h3 class="languages">Languages</h3>
...
             </div>
             <img src="./images/Separator-Bottom.png" alt="" class="separator" />
             <div class="event18">
               <h3>Skills</h3>
               <button type="submit" class="dropdown">
-                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                <img src="./images/Disabled.png" alt="" />
               </button>
             </div>
             <img src="./images/Separator-Bottom.png" alt="" class="separator" />
```
Comment (Lizdev-05) [inline index.html:346]: Kindly check the image paths, the necessary icons are not here, You can check the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for more understanding.
**Your design**
arrow not

**Figma**
fig

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @obibaadoma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights
- All linters checks are green ✔️
- Descriptive PR title and summary✔
- The repository is documented in a professional way 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (obibaadoma) [reply index.html]: fixed the design

File: index.html (hunk 3 above)
Comment (obibaadoma) [reply index.html:346]: resolved to meet figma design

File: index.html [hunk 4]
```diff
@@ -254,10 +257,10 @@ <h3>2015</h3>
   <section class="contact-me">
     <div class="head-section-co">
       <div id="head-container-co">
-        <h1 class="head-co">
-          About, <br />
+        <h2 class="head-co">
+          About <br />
           Myself
```
Comment (1GORDON) [inline index.html:262]: Great job @obibaadoma 🤝

- [ ] [Optional] As stated in the project requirements, kindly align the elements of the contact me section with flexbox.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @obibaadoma ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions

- [ ] You may always write your commit messages in an imperative tense something like `Change Button and Image` instead of `Changes to Button and Image`.
- [ ] You may also fix the MIT license link to make your Readme file more professional. Right now the link appears to be broken.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**